wallets/react/src/types.ts

```diff
@@ -23,7 +27,11 @@ export type ConnectResult = {
 export type Providers = { [type in WalletType]?: any };
 
 export type ProviderContext = {
-  connect(type: WalletType, network?: Network): Promise<ConnectResult>;
+  connect(
+    type: WalletType,
+    network?: Network,
```

Comment:
To be more compatible with upcoming v2, Could you change `network: Network` to `network: NamespaceAndNetwork`? You can see the type here: https://github.com/rango-exchange/rango-client/pull/623/files#diff-64a6b181d2c51d0a93de41b2428c6442eef2627be9aef4294b3bdd41a6e1f4b1R143

Note:  You don't need DISCOVERY_MODE.

Reply:
Considering that v2 is not merged right now and it is not known that when we can merge it, I think we can not add anything about namespace because it is an unknown concept for other developers and can be confusing